src/components/v2/Table/Head.tsx

```diff
@@ -6,7 +6,7 @@ import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
 import Box from '@mui/material/Box';
 import { visuallyHidden } from '@mui/utils';
-import { Icon } from '../Icon';
+import { Icon } from 'components/v2/Icon';
```

Comment:
I think it is better to avoid using v2 in imports because at some point we will remove it and if we export and import everything from `components/index` then we won't have to change any imports

In this case, as a relative import it also wouldn't change when we get rid of `v2`